src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/changeset/RelationChangeSet.java

```diff
@@ -57,16 +57,23 @@
     private final Set<Long> deletedLines;
     private final Set<Long> deletedRelations;
 
+    // Relations that were sliced will be deleted and replaced by two or more new Relations. We need
```

Comment:
Nice comments! Really easy to read and understand!

Reply:
Thanks!